### PR TITLE
Add missing logs from the gcp demo

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Fixes https://github.com/ballerina-guides/gcp-microservices-demo/issues/44
Fixes https://github.com/ballerina-guides/gcp-microservices-demo/issues/39

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: frontend/service.bal [hunk 1]
```diff
@@ -207,7 +212,7 @@ service / on new http:Listener(9098) {
         response.setJsonPayload({
             user_currency: [request.currency, currencyLogo(request.currency)],
             currencies: supportedCurrencies,
-            cart_size: cart.items.length(),
+            cart_size: check getCartSize(cart),
```
Comment (xlight05) [inline frontend/service.bal:215]: Is there any reason for this change?

File: productcatalog/utils.bal [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import ballerina/log;
```
Comment (xlight05) [inline productcatalog/utils.bal:1]: Import before licence

File: frontend/service.bal (hunk 1 above)
Comment (dilanSachi) [reply frontend/service.bal:215]: 2 items from one products counted as 1 in our previous impl

File: productcatalog/utils.bal (hunk 2 above)
Comment (dilanSachi) [reply productcatalog/utils.bal]: Fixed https://github.com/ballerina-guides/gcp-microservices-demo/pull/52/commits/6a2f0a71c84b6551a3b589b42edeadffa647c1d8

**Review by xlight05 — APPROVED** [review]

**Pull request merged**